frappe/core/doctype/activity_log/activity_log.py

```diff
@@ -49,5 +49,5 @@ def clear_activity_logs(days=None):
 		days = 90
 	doctype = DocType("Activity Log")
 	frappe.db.delete(doctype, filters=(
-		doctype.creation < PseudoColumn(f"({Now() - Interval(days=days)})")
+		doctype.creation < (Now() - Interval(days=days))
 	))
```

Comment:
This changes the original query

Original query ->
`delete from "tabActivity Log" where \t\tcreation< (NOW() - INTERVAL '10' DAY)`

Query obtained using pseudocolumn ->
`DELETE FROM "tabActivity Log" WHERE "creation"<(NOW()-INTERVAL '10 DAY')`

Query when pseudocolumn is removed ->
`DELETE FROM "tabActivity Log" WHERE "creation"<NOW()-INTERVAL '10 DAY'`

Reply:
@Aradhya-Tripathi can you tell me how to verify the raw SQL query generated by qb?

Reply:
You can pass ```run=False``` as a kwarg to all queries going through ```def sql```  to just return the query generated

Reply:
@Aradhya-Tripathi Yeah, you were right. Thanks for pointing this out. Can you check why the system was throwing syntax errors then?
Also, my query seems to be working fine (as intended). can you verify that? 😅

Reply:
Hey so your query isn't wrong ie it won't give a syntax error it just differs in logic diff -> `creation<(Now() - Interval(days = days))`